### PR TITLE
Update connected registry disk error description

### PR DESCRIPTION
**Pull request by jaysterp**

### CODE REVIEW

File: docs/preview/connected-registry/connected-registry-error-codes.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Sample status detail:
     {
       "code": "DiskError",
       "correlationId": "73a46395-b89b-49c7-5621-d54e8b1574b5",
-      "description": "There is not enough space on the disk.",
+      "description": "No space left on device.",
```
Comment (toddysm) [inline docs/preview/connected-registry/connected-registry-error-codes.md:110]: Shouldn't this read "Insufficient disk space on the device"?

File: docs/preview/connected-registry/connected-registry-error-codes.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ This status `description` indicates that the connected registry was unable to wr
 
 *Potential solution:* Ensure that the host storage path used to run the connected registry container gives sufficient access to the container user. In the sample above, this path is `/var/acr/data/registry`. Update the permissions of the host system directory so that the user profile for your container has read, write, and execute access. By default, docker containers run as root. If the container is run as a non-root user, please ensure that user has the above permissions.
 
-### No Storage Available On Disk
+### No Storage Available On Device
```
Comment (toddysm) [inline docs/preview/connected-registry/connected-registry-error-codes.md:101]: In the spirit of being consistent with the wording below, I would say "Insufficient disk space" for the title also.

File: docs/preview/connected-registry/connected-registry-error-codes.md (hunk 1 above)
Comment (jaysterp) [reply docs/preview/connected-registry/connected-registry-error-codes.md:110]: @toddysm "No space left on device" is the standard Linux error message when a file tries to be created when the device is out of disk space

File: docs/preview/connected-registry/connected-registry-error-codes.md (hunk 2 above)
Comment (jaysterp) [reply docs/preview/connected-registry/connected-registry-error-codes.md]: Please see update to header name.

**Review by toddysm — APPROVED** [review]

**Pull request merged**